backend/src/main/java/edonymyeon/backend/thumbs/domain/Thumbs.java

```diff
@@ -66,4 +67,11 @@ public void up() {
         }
         updateThumbsType();
     }
+
+    public void down() {
+        if (!isUp()) {
```

Comment:
이미 존재하는 isDown()이라는 메서드를 쓰면 더 명확히 보일 것 같아용
```suggestion
    public void down() {
        if (isDown()) {
```

Reply:
수정하였습니다!!